hosted/gke/p1/p1_provisioning_test.go

```diff
@@ -195,6 +195,64 @@ var _ = Describe("P1Provisioning", func() {
 			_, err = helper.AddNodePool(cluster, ctx.RancherAdminClient, 1, "WINDOWS_LTSC_CONTAINERD", true, true)
 			Expect(err).To(BeNil())
 		})
+
+		It("should not be able to update the only non-windows nodepool to windows", func() {
+			testCaseID = 263
```

Comment:
```suggestion
			testCaseID = 34
```

Reply:
The testCaseID is alright, but I have modified the test title to be similar to the one on Qase.